distribution/zip/pom.xml

```diff
@@ -60,7 +60,11 @@
                         </goals>
                         <configuration>
                             <target if="${run}">
-                                <ant antfile="${elasticsearch.integ.antfile}" target="start-foreground"/>
+                                <!-- use conventional port numbers -->
+                                <ant antfile="${elasticsearch.integ.antfile}" target="start-foreground">
+                                    <property name="integ.http.port" value="9200"/>
```

Comment:
Can't developers run it using default testing ports?

```xml
<integ.http.port>9400</integ.http.port>
<integ.transport.port>9500</integ.transport.port>
```

I mean that if a dev really want to change the running testing port, he can for sure add a profile in his own maven settings.xml file, right?